src/material/tabs/paginated-tab-header.ts

```diff
@@ -246,6 +246,35 @@ export abstract class MatPaginatedTabHeader
     });
   }
 
+  /** A method responsible for sending any change that could affect layout about
+   * items to subscribers.
+   */
+  _getItemChanges() {
+    const itemContainerItemChanged = new Subject<void>();
+
+    // Check to see if ResizeObserver is supported and if not, stub the function
+    // out so that it does nothing if not supported
+    const observer =
+      typeof ResizeObserver === 'function'
+        ? new ResizeObserver(() => {
+            itemContainerItemChanged.next();
+          })
+        : {
+            observe: () => {},
+          };
+
+    return merge(this._items.changes, itemContainerItemChanged).pipe(
+      startWith(this._items),
+      map(() => {
+        for (const item of this._items.toArray()) {
+          this._ngZone.run(() => {
```

Comment:
This still hasn't been changed to `runOutSizeAngular`.

Reply:
Sorry, I forgot to push.